proto/types/canonical.proto

```diff
@@ -29,9 +29,9 @@ message CanonicalProposal {
 
 message CanonicalVote {
   SignedMsgType             type      = 1;  // type alias for byte
-  int64                     height    = 2 [(gogoproto.moretags) = "binary:\" fixed64 \""];
-  int64                     round     = 3 [(gogoproto.moretags) = "binary:\" fixed64 \""];
-  CanonicalBlockID          block_id  = 5 [(gogoproto.nullable) = false, (gogoproto.customname) = "BlockID"];
-  google.protobuf.Timestamp timestamp = 6 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
-  string                    chain_id  = 7 [(gogoproto.customname) = "ChainID"];
+  sfixed64                  height    = 2 [(gogoproto.moretags) = "binary:\"fixed64 \""];
+  sfixed64                  round     = 3 [(gogoproto.moretags) = "binary:\"fixed64 \""];
```

Comment:
Is the `"binary:\"fixed64 \""` still needed here?

Reply:
they can not be negative But I believe we should change them to uints once we change the height and round throughout the repo. I believe this would come in 0.35.

I removed fixed:64 but would still like confirmation from juan on how ledger things work. 

I wonder who really uses a ledger for validator signing, maybe this feature shouldn't be supported in tmkms?